app/assets/javascripts/tylium/modules/local_auto_save.js

```diff
@@ -45,7 +56,15 @@ document.addEventListener("turbolinks:load", function() {
 
     function getData(formInputs) {
       var reducer = function(hash, input) {
-        hash[input.name] = input.value;
+        // Check if name is an array, i.e. checkboxes
+        if (input.name.slice(-2) == "[]") {
+          var a = hash[input.name] = Array.from(form.querySelectorAll(`[name="${input.name}"]:checked`)).map(function(input) {
```

Comment:
@sean-yeoh can you split this into lines that's more readable?

Reply:
I've updated the whole local auto save module to a better versiion.